modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc

```diff
@@ -16,7 +16,7 @@ Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD i
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the Red Hat OpenShift GitOps Operator is installed.
 . Navigate to the {rh-app-icon} menu -> *OpenShift GitOps* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
 . Obtain the password for the Argo CD instance:
-.. Navigate to the *Developer* perspective of the web console. A list of available projects is displayed.
+.. In the left panel of the console, use the perspective switcher to switch to the *Developer* perspective. The *Topology* view with a list of available projects in your cluster is displayed.
 .. Navigate to the `openshift-gitops` project.
```

Comment:
@DebarghoGhosh I did not get the list of all the projects in the Topology view. Hence I included the following step as Optional:
.. Optional: If you do not see the list of available projects, use the *Project* drop-down list and click *All Projects*. A list of all the projects in your cluster is displayed.

I am not sure if it is just me. But this optional step helps the user if he/she is also experiencing the same problem. I was wondering why is it not appearing for me and figured it out on my own. Just double-check and do what is useful and appropriate for the user.

Reply:
@Srivaralakshmi for me the All Projects drop-down was selected by default and the list of all the projects appear in the Topology. So not sure whether to add or skip. @Preeticp please check and confirm.

Reply:
The state in your clusters is what you see based on who (other folks who are using the same cluster) used what earlier on the cluster, so that explains the difference you both see. We can't predict from where our users are coming. So I would suggest to ask the developers as to what is the default behavior for a first time user - Vikram from the ODC team in forum devconsole channel/or maybe Regina.
A workaround would be, 1. Remove the sentence - The *Topology* view with a... and change the second step to: Navigate to the `openshift-gitops` project./Use the Project drop-down list and select the `openshift-gitops` project.

Reply:
@Preeticp @Srivaralakshmi I have made the changes as suggested. Please review